### PR TITLE
[4.13] Add support to Shielded VMs on GCP

### PR DESCRIPTION
**Pull request by jianli-wei**
1. The 4.13 epic: [CORS-2424 Add support to Shielded VMs on GCP](https://issues.redhat.com/browse/CORS-2424)
2. Add workflow `cucushift-installer-rehearse-gcp-ipi-secureboot` and the provision chain
3. Add periodic jobs of `gcp-ipi-secureboot-p2-f7` and `gcp-ipi-secureboot-p2-f28-destructive`

### CODE REVIEW

**Comment by jianli-wei** [comment]
/pj-rehearse periodic-ci-openshift-verification-tests-master-installer-rehearse-4.13-installer-rehearse-gcp

**Comment by jianlinliu** [comment]
cc @liangxia for approval

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly.yaml [hunk 1]
```diff
@@ -1893,6 +1893,26 @@ tests:
     test:
     - chain: openshift-e2e-test-qe-destructive
     workflow: cucushift-installer-rehearse-gcp-ipi-realtime-kernel
+- as: gcp-ipi-secureboot-p2-f7
```
Comment (liangxia) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly.yaml:1896]: I think `gcp-ipi-sec*` sorts after `gcp-ipi-sdn*`

Comment (jianli-wei) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly.yaml]: @liangxia It seems not well-sorted with existing sections of "- as: gcp", anyway I moved the 2 new sections after `gcp-ipi-sdn*` as you suggested, thanks! 
```
[jiwei@jiwei release]$ grep "\- as: gcp" ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-
4.13__amd64-nightly.yaml -n
1638:- as: gcp-ipi-baselinecaps-v411-additionalcaps-p2-f7
1648:- as: gcp-ipi-baselinecaps-v411-additionalcaps-p3-f28-destructive
1658:- as: gcp-ipi-cco-manual-workload-identity-p1-f4
1668:- as: gcp-ipi-cco-manual-workload-identity-p1-f28-destructive
1678:- as: gcp-ipi-disconnected-private-sts-p2-f7
1690:- as: gcp-ipi-disconnected-private-p3-f28
1702:- as: gcp-ipi-disconnected-private-p3-f28-destructive
1714:- as: gcp-ipi-disconnected-workers-rhel8-p2-f7
1726:- as: gcp-ipi-disconnected-workers-rhel8-p2-f28-destructive
1738:- as: gcp-ipi-xpn-p2-f7
1748:- as: gcp-ipi-ovn-ipsec-p1-f4
1759:- as: gcp-ipi-ovn-ipsec-p1-f28-destructive
1769:- as: gcp-ipi-ovn-mtu-migrate-p2-f7
1779:- as: gcp-ipi-ovn-mtu-migrate-p2-f28-destructive
1789:- as: gcp-ipi-p3-f28-long-duration-part3-3
1801:- as: gcp-ipi-private-disasterrecovery-p2-f7
1813:- as: gcp-ipi-proxy-etcd-encryption-p2-f7
1823:- as: gcp-ipi-proxy-etcd-encryption-p2-f28-destructive
1834:- as: gcp-ipi-proxy-private-p1-f4
1844:- as: gcp-ipi-proxy-private-p1-f28-destructive
1855:- as: gcp-ipi-proxy-techpreview-p3-f28
1865:- as: gcp-ipi-proxy-techpreview-p3-f28-destructive
1876:- as: gcp-ipi-realtime-kernel-p2-f7
1886:- as: gcp-ipi-realtime-kernel-p2-f28-destructive
1896:- as: gcp-ipi-sdn-migration-ovn-p2-f7
1908:- as: gcp-ipi-sdn-p1-f7
1918:- as: gcp-ipi-sdn-p3-f28-destructive
1928:- as: gcp-ipi-secureboot-p2-f28-destructive
1938:- as: gcp-ipi-secureboot-p2-f7
1948:- as: gcp-ipi-workers-rhel8-fips-p2-f7
1958:- as: gcp-ipi-workers-rhel8-fips-p3-f28-destructive
1969:- as: gcp-upi-baselinecaps-v411-p2-f7
1979:- as: gcp-upi-baselinecaps-v411-p3-f28-destructive
1989:- as: gcp-upi-p3-f28
1999:- as: gcp-upi-p3-f28-destructive
2009:- as: gcp-upi-private-xpn-ingress-glb-p2-f7
2019:- as: gcp-upi-private-xpn-ingress-glb-p3-f28-destructive
2029:- as: gcp-upi-private-xpn-sdn-p2-f7
2039:- as: gcp-upi-private-xpn-sdn-p3-f28-destructive
2049:- as: gcp-upi-xpn-p2-f14
2059:- as: gcp-upi-xpn-p3-f28-destructive
[jiwei@jiwei release]$

Comment (liangxia) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly.yaml]: Good catch.
Fix in https://github.com/openshift/release/pull/36207

**Pull request merged**